pymeasure/instruments/ni/virtualbench.py

```diff
@@ -1274,15 +1250,15 @@ def configure_analog_pulse_width_trigger(self, trigger_source,
                 trigger_source, trigger_polarity, trigger_level,
                 comparison_mode, lower_limit, upper_limit, trigger_instance)
 
-        def configure_digital_pulse_width_trigger(self, trigger_source,
-                                                  trigger_polarity,
-                                                  comparison_mode,
-                                                  lower_limit, upper_limit,
-                                                  trigger_instance):
-            ''' Configures a trigger to activate on the specified source when
-            the digital edge reaches the specified levels within a specified
-            window of time.
-            '''
+        # def configure_digital_pulse_width_trigger(self, trigger_source,
```

Comment:
Please delete this instead of only commenting out.

Reply:
you are right. removed the methods which were commented-out. added some information on the missing methods in the classes docstring instead. hope this is fine.